contracts/governance/Staking/Staking.sol

```diff
@@ -302,4 +369,36 @@ contract Staking is WeightedStaking{
         allUnlocked = true;
         emit TokensUnlocked(SOVToken.balanceOf(address(this)));
     }
+
+    /**
+     * @notice Gets list of stakes for `account`
+     * @param account The address to get stakes
+     * @return The arrays of dates and stakes
+     */
+    function getStakes(address account) public view returns (uint[] memory dates, uint96[] memory stakes) {
```

Comment:
maybe make this external? it's only for frontend usage, isn't it?

Reply:
yes, done